util/wallaby.cred.loader.js

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const credentialsPromise = require('@aws-sdk/credential-provider-node').defaultProvider()()
+
+module.exports = async function (source, map) {
+  console.log('fuck you')
```

Comment:
Might want to fix this up...

Reply:
Yes, thanks.  I was up late, that is a mistake.